x-pack/plugins/cases/public/components/all_cases/columns.tsx

```diff
@@ -281,6 +312,28 @@ export const useCasesColumns = ({
         return getEmptyTagValue();
       },
     },
+    ...(isSelectorView
+      ? [
+          {
+            render: (theCase: Case) => {
+              if (theCase.id != null) {
+                return (
+                  <EuiButtonEmpty
```

Comment:
According to @hbharding we should use `<EuiButton size="small" filled>Select</EuiButton>` instead of an empty button, since it's the primary action and we don't want to mistaken it as a link.